step-templates/import-databricks-workbooks.json

```
@@ -0,0 +1,76 @@
+  },
+  "Parameters": [
+    {
+      "Id": "b8cd5d73-29d9-4ba2-bdfa-86cc1316a16f",
+      "Name": "DeployDataBricksWorkBookPackage",
+      "Label": "DataBricks WorkBook Package",
+      "HelpText": "The Databricks workbook package",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Package"
+      }
+    },
+    {
+      "Id": "f81b9ccb-beea-4d8f-a049-ee9ea6da643e",
+      "Name": "DataBricksInstanceUri",
+      "Label": "Databricks Instance Uri",
+      "HelpText": "The Databricks Instance URL",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "6c1aed25-f9a6-4c1f-b8e9-d50cc8669f2d",
+      "Name": "AccessToken",
+      "Label": "Databricks Access Token",
+      "HelpText": "The access token to authenticate against the Databricks instance",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Sensitive"
+      }
+    },
+    {
+      "Id": "c95af088-180b-419e-8c44-ce3fb4d96f57",
+      "Name": "DatabrickImportFolder",
```

Not essential but it would be good for consistency to have this parameter include an additional `s` to become `DatabricksImportFolder` (you'll also need to update any usages internally in your script if you do update it)
```suggestion
      "Name": "DatabricksImportFolder",
```